src/widgets/Text.vue

```diff
@@ -22,16 +30,21 @@
       :name="field.name"
       :aria-invalid="hasErrors ? 'true' : null"
       :aria-describedby="field.options.hasOwnProperty('help') ? `${getFieldId()}-help` : null"
-      v-on:keypress.enter.prevent="submitCallback"
-      v-bind:value="value"
-      v-on:input="value = $event.target.value">
+      :data-qa-locator="qaLocator ? `${qaLocator}-input` : null"
+      @keypress.enter.prevent="submitCallback"
+      :value="value"
+      @input="value = $event.target.value">
```

Comment:
Though can this one be reformatted to void/XHTML stylee, for consistency?

```suggestion
      @input="value = $event.target.value"
    />
```